django_crate/backend/schema.py

```diff
@@ -80,109 +98,65 @@ def create_model(self, model):
                 definition += " %s" % col_type_suffix
             params.extend(extra_params)
 
-            # FK
-            """NOT SUPPORTED
-
-            if field.rel and field.db_constraint:
-                to_table = field.rel.to._meta.db_table
-                to_column = field.rel.to._meta.get_field(field.rel.field_name).column
-                if self.connection.features.supports_foreign_keys:
-                    self.deferred_sql.append(
-                        self.sql_create_fk % {
-                            "name": self._create_index_name(model, [field.column], suffix="_fk_%s_%s" % (to_table, to_column)),
-                            "table": self.quote_name(model._meta.db_table),
-                            "column": self.quote_name(field.column),
-                            "to_table": self.quote_name(to_table),
-                            "to_column": self.quote_name(to_column),
-                        }
-                    )
-                elif self.sql_create_inline_fk:
-                    definition += " " + self.sql_create_inline_fk % {
-                        "to_table": self.quote_name(to_table),
-                        "to_column": self.quote_name(to_column),
-                    }
-            """
             # Add the SQL to our big list
             column_sqls.append("%s %s" % (
                 self.quote_name(field.column),
                 definition,
             ))
-            """NOT SUPPORTED
-            # Autoincrement SQL (for backends with post table definition variant)
-            if field.get_internal_type() == "AutoField":
-                autoinc_sql = self.connection.ops.autoinc_sql(model._meta.db_table, field.column)
-                if autoinc_sql:
-                    self.deferred_sql.extend(autoinc_sql)
-            """
-
-        # CRATE OPTIONS
+
+        # Make the table
+        sql = self.sql_create_table % {
+            "table": self.quote_name(model._meta.db_table),
+            "definition": ", ".join(column_sqls),
+            "clustered_by": self._clustered_by_sql(model, params),
+            "partitioned_by": self._partitioned_by_sql(model, params),
+            "settings": self._table_settings_sql(model, params),
+        }
+
+        # Prevent using [] as params, in the case a literal '%' is used in the definition
+        self.execute(sql, params or None)
+        # wait for shards to start
+        # let's do this optimistically ...
+        time.sleep(2)
```

Comment:
Wishlist: "someday", I hope we can fix the behavior of CREATE TEST DATABASE w/r/t crate tables in django unit tests, and on that day, this time.sleep(2) will be really annoying; for now, it is pretty much a moot point, leave it